CMakeLists.txt

```diff
@@ -17,3 +18,7 @@ ENDIF()
 IF(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
     add_subdirectory(cntp-windeploy)
 ENDIF()
+
+IF(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
+    add_subdirectory(cntp-trappstream)
+ENDIF()
```

Comment:
We'd need to remove the `if(LINUX)` if we want to implement the JSON generation on other platforms. The program works as-is on Windows and Mac anyway

Reply:
Implemented